util/src/main/java/io/kubernetes/client/util/Config.java

```diff
@@ -26,6 +26,8 @@
 
 import javax.net.ssl.KeyManager;
 
+import org.apache.commons.codec.binary.Base64;
```

Comment:
unused import. is there a way to detect these automatically in a github check? like a linter?